pockyt/client.py

```diff
@@ -165,9 +166,17 @@ def _get(self):
             'title': item.get('resolved_title'),
             'link': item.get('resolved_url'),
             'excerpt': item.get('excerpt'),
-            'tags': item.get('tags'),
+            'tags': self._displayTags(item.get('tags')),
         } for item in items.values())
 
+    def _displayTags(self, item_tags):
```

Comment:
Should be `_process_tags` to match style and clarify the verb.

Reply:
Makes sense. Done!